cads_adaptors/adaptors/mars.py

```diff
@@ -21,7 +21,15 @@ def execute_mars(request: Request, target="data.grib"):
     user_id = 0
     env["MARS_USER"] = f"{namespace}-{user_id}"
 
-    subprocess.run(["/usr/local/bin/mars", "r"], check=True, env=env)
+    output = subprocess.run(
+        ["/usr/local/bin/mars", "r"], check=False, env=env, capture_output=True
+    )
+    context.stdout = output.stdout.decode()
+    context.stderr = output.stderr.decode()
+    context.user_visible_log = output.stdout.decode()
+
+    if output.returncode:
+        raise RuntimeError("The MARS adaptor has crashed.")
```

Comment:
ATM it isn't possible to raise a custom exception defined in cads-adaptors because the broker doesn't have it installed. In case we want to define new custom exceptions for the adaptors we need to install cads-adaptors in the broker because the broker needs import the exceptions in order to correctly parse them.